gammapy/datasets/utils.py

```diff
@@ -67,3 +70,83 @@ def get_nearest_valid_exposure_position(exposure, position=None):
     if not position:
         position = mask_exposure.geom.center_skydir
     return mask_exposure.mask_nearest_position(position)
+
+
+def split_dataset(dataset, width, margin, split_templates=True):
+    """Split dataset in multiple non-overlapping analysis regions.
+
+    Parameters
+    ----------
+    dataset : `~gammapy.datasets.Dataset`
+        Dataset to split
+    split_templates : bool, optional
```

Comment:
Maybe the list of parameters in the docstring could be ordered like in the function... A detail